one-node-ce/README.md

```diff
@@ -91,11 +95,13 @@ After `Dockerfile_<distro>` installs the RPM or DEB file, it runs `tools/cleanup
 
 # Testing with ./run_tests.sh
 
+Once you have built your container, you can test it using the `./run_tests.sh` script (or by running `make test`).
+
 The `./run_tests.sh` script verifies that the container can execute Vertica and some of the additional libraries. This script requires a [local copy of the vsql client](#getting-a-local-copy-of-vsql). You can run the script directly, or you can run it as `make test`.
 
 Before you test your container, you must stop any existing Vertica server (container or otherwise) on your test system because the `./run_tests.sh` script uses the normal Vertica port number.
 
-The test creates a new container with a unique tab, and volume. Because the test sets up the optional libraries and creates the VMart database, creating a new container can take as long as three minutes.
+The test uses your image to create a new container with a unique tab, and volume. Because the test sets up the optional libraries and creates the VMart database, creating a new container can take as long as three minutes.
```

Comment:
"new container with a unique tab," what is a tab?

Reply:
should be "tag"